tests/store/api/conftest.py

```diff
@@ -180,6 +180,102 @@ def fixture_rml_store(store: Store, helpers: StoreHelpers) -> Store:
     return store
 
 
+@pytest.fixture(name="case_id_with_single_sample")
+def case_id_with_single_sample():
+    return "screamingtiger"
```

Comment:
Would it be possible to use the case_id fixture (inherited from earlier conftext) here? Or do you need specific case_ids and sample_ids?

Reply:
I thought it would make it easier to understand the tests if I created specific fixtures with descriptive names. So "case_id_with_single_sample" is easier to use and understand compared to "case_id". So I would like to keep these, but I understand that it is better to reuse existing fixtures if possible. Is it ok in this instance?

Reply:
Sounds good with descriptive names. How about returning the case_id fixture from this one?